Tasks/google-play-release/task.json

```diff
@@ -170,6 +178,15 @@
             "required": false,
             "helpMarkDown": "Paths to additional APK files you want to publish to the specified track (e.g. an x86 build). Wildcards can be used. For example, _\\*\\*/\\*.apk_ to match all APK files, in any directory."
         },
+        {
+            "name": "shouldPickObbFileForAdditonalApks",
```

Comment:
I'm a bit confused with this.  Do you expect there to ever be a condition where `shouldPickObbFile` is used without `shouldPickObbFileForAdditonalApks` or vice-versa?  If not, should we have the two options?

Reply:
Yes, I did expect so. According to the design, there should be an option to pick obb for any apk. So I think 2 options would make sense